app/javascript/src/Navigation/components/PF4NavProxy.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+
+import 'Navigation/styles/PF4NavProxy.scss'
+
+const Nav = ({id, children}) => (
+  <nav id={id} className="pf-c-nav" aria-label="Global">
+    {children}
+  </nav>
+)
+
+const NavExpandable = ({id, title, isExpanded, isActive, children}) => {
+  const [expanded, setExpanded] = useState(isExpanded)
+
+  function onItemClick () {
+    setExpanded(expanded => !expanded)
+  }
+
+  return (
+    <li className={`pf-c-nav__item pf-m-expandable ${expanded ? 'pf-m-expanded' : ''} ${isActive ? 'pf-m-current' : ''}`}>
+      <a href="#" className="pf-c-nav__link" id={id} aria-expanded="true" onClick={onItemClick}>
+        {title}
+        <span className="pf-c-nav__toggle">
+          <svg style={{ verticalAlign: '-0.125em' }} fill="currentColor" height="1em" width="1em" viewBox="0 0 256 512" aria-hidden="true" role="img">
```

Comment:
maybe name this svg in a `const` in order to know what it is

Reply:
Keep in mind this file is not meant to be reviewed nor modified since it's literally a copy-🍝 of the Patternfly markup. It *has to be* exactly identical as per Patternfly documentation.

That being said, OK I will extract this into a constant.